src/main/resources/view/EventDetailPanel.fxml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.text.Text?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.control.ScrollPane?>
+
+
+<?import javafx.scene.control.Separator?>
+
+
+
+<AnchorPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            styleClass="detail-box"
+            VBox.vgrow="ALWAYS">
+
+    <ScrollPane AnchorPane.topAnchor="0" AnchorPane.bottomAnchor="0"
+                AnchorPane.leftAnchor="0" AnchorPane.rightAnchor="0" style="-fx-background-color: transparent;" fitToHeight="true" fitToWidth="true">
+
+        <VBox id="eventDetailView" fx:id="eventDetailView" styleClass="detail-view"
+              xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/22">
+
+            <!-- Event Title -->
+            <Label fx:id="title" styleClass="detail-header" text="\$title" />
+
+            <!-- Event Info Section -->
+            <VBox spacing="15">
+                <HBox spacing="15" alignment="CENTER_LEFT">
+                    <HBox spacing="15" prefWidth="150" alignment="CENTER_LEFT">
+                        <ImageView>
+                            <image>
+                                <Image url="@/images/calendar.png" />
+                            </image>
+                        </ImageView>
+                        <Text styleClass="detail-contents"> Date </Text>
+                    </HBox>
+                    <Label fx:id="date" styleClass="detail-contents" text="\$date"/>
+                </HBox>
+
+                <!-- Attendees Section -->
+                <VBox spacing="20">
```

Comment:
Can we increase the top margin of the attendees section? So that there's more spacing from the event details section

Reply:
Sure, makes sense.